WooCommerce/Classes/Extensions/UserDefaults+Woo.swift

```diff
@@ -50,6 +50,7 @@ extension UserDefaults {
 
         // Background Task Refresh
         case latestBackgroundOrderSyncDate
+        case lastBackgroundRefreshTime
```

Comment:
nit: maybe this could indicate that it's the "completion" time to be more exact.
```suggestion
        case lastBackgroundRefreshCompletionTime
```